cdap-ui/app/cdap/components/Replicator/Create/Content/Assessment/TablesAssessment/Mappings/index.tsx

```diff
@@ -173,7 +176,7 @@ const MappingsView: React.FC<IMappingsProps> = ({
   const [loading, setLoading] = React.useState(true);
   const [rerunAssessment, setRerunAssessment] = React.useState(false);
 
-  function assessTable() {
+  function assessTable(updatedColumns?) {
```

Comment:
TIL: you can use `?` to specify optional field without a type!. Nice, but looks like incomplete ternary operator or null coalescing 😄

Reply:
with the new null coalescing, the code make it seemed like the programmer is unsure about everything with question marks everywhere 😂